lib/release_notes/github_api.rb

```diff
@@ -62,11 +62,13 @@ def find_content(changelog_file)
       return nil
     end
 
-    def update_changelog(file, summary, changelog_content, branch: "master")
+    def update_changelog(file, summary, changelog_content, changelog_repo: nil, branch: "master")
+      @client.update_contents(changelog_repo, file.path, summary, file.sha, changelog_content, branch: "master") if changelog_repo
       @client.update_contents(@repo, file.path, summary, file.sha, changelog_content, branch: "master")
     end
 
-    def create_content(file, changelog_content, message: "Creating Changelog", branch: "master")
+    def create_content(file, changelog_content, message: "Creating Changelog", changelog_repo: nil, branch: "master")
+      @client.create_content(changelog_repo, file, message, changelog_content, branch: branch) if changelog_repo
```

Comment:
This creates the changelog in the repo we set, but it also creates a changelog in the base app. I think this makes sense, that we would want to keep track of all of our clients in the base app, but do you think this is `unexpected behaviour` that it creates two changelogs. Should I make this an option that the user passes in `include_changelog_in_base: false` defaulting it to false?

Reply:
Or maybe it should be an array of where you want the changelog to be created. `changelog_repos`. By default, we will only create the changelog in the repo you are running the release notes on. If you want it in multiple places, you have to state [current_repo, extra_repo, ...]